### PR TITLE
Update for the JOSS paper editor review

### PR DESCRIPTION
**Pull request by neillu23**
## What?

1. Add references to the actual model architecture papers as the paper mentions that espnet-se++ adds "several new models, loss functions, and training recipes". 
2. Add DOIs for the references. This is required for a JOSS publication
3. Modify title={Onssen: an open-source ...} to title={{ONSSEN}: an open-source ...}

## Why?
To address the comments from editor review  https://github.com/espnet/espnet/issues/5399  for the JOSS paper "Software Design and User Interface of ESPnet-SE++: Speech Enhancement for Robust Speech Processing"

### CODE REVIEW

**Comment by faroit** [comment]
@Emrys365 Hi 👋. Would it be possible to review this PR soon so that we can continue with the [JOSS publication](https://github.com/espnet/espnet/issues/5399)

File: doc/paper/espnet-se++/paper.bib [hunk 1]
```diff
@@ -7,17 +8,156 @@ @inproceedings{Li:2021
...
+@inproceedings{Li:2022,
+    doi = {10.1109/icassp43922.2022.9746372},
+    url = {https://doi.org/10.1109%2Ficassp43922.2022.9746372},
+    year = 2022,
+    month = {may},
+    publisher = {{IEEE}},
+    author = {Li, C. and Yang, L. and Wang, W. and Qian, Y.},
+    title = {Skim: Skipping Memory Lstm for Low-Latency Real-Time Continuous Speech Separation},
+    booktitle = {{ICASSP} 2022 - 2022 {IEEE} International Conference on Acoustics, Speech and Signal Processing ({ICASSP})}
+}
```
Comment (Emrys365) [inline doc/paper/espnet-se++/paper.bib:58]:
```suggestion
@inproceedings{Li:2022,
    doi = {10.1109/icassp43922.2022.9746372},
    url = {https://doi.org/10.1109%2Ficassp43922.2022.9746372},
    pages={681--685},
    year = 2022,
    month = {may},
    publisher = {{IEEE}},
    author = {Li, C. and Yang, L. and Wang, W. and Qian, Y.},
    title = {{SkiM}: Skipping Memory Lstm for Low-Latency Real-Time Continuous Speech Separation},
    booktitle = {2022 {IEEE} International Conference on Acoustics, Speech and Signal Processing ({ICASSP})}
}
```

File: doc/paper/espnet-se++/paper.bib [hunk 2]
```diff
@@ -7,17 +8,156 @@ @inproceedings{Li:2021
...
+@inproceedings{Dang:2022,
+    doi = {10.1109/icassp43922.2022.9746171},
+    url = {https://doi.org/10.1109%2Ficassp43922.2022.9746171},
+    year = 2022,
+    month = {may},
+    publisher = {{IEEE}},
+    author = {Dang, F. and Chen, H. and Zhang, P.},
+    title = {{DPT}-{FSNet}: Dual-Path Transformer Based Full-Band and Sub-Band Fusion Network for Speech Enhancement},
+    booktitle = {{ICASSP} 2022 - 2022 {IEEE} International Conference on Acoustics, Speech and Signal Processing ({ICASSP})}
+}
```
Comment (Emrys365) [inline doc/paper/espnet-se++/paper.bib:69]:
```suggestion
@inproceedings{Dang:2022,
    doi = {10.1109/icassp43922.2022.9746171},
    url = {https://doi.org/10.1109%2Ficassp43922.2022.9746171},
    pages={6857--6861},
    year = 2022,
    month = {may},
    publisher = {{IEEE}},
    author = {Dang, F. and Chen, H. and Zhang, P.},
    title = {{DPT}-{FSNet}: Dual-Path Transformer Based Full-Band and Sub-Band Fusion Network for Speech Enhancement},
    booktitle = {2022 {IEEE} International Conference on Acoustics, Speech and Signal Processing ({ICASSP})}
}
```

File: doc/paper/espnet-se++/paper.bib [hunk 3]
```diff
@@ -7,17 +8,156 @@ @inproceedings{Li:2021
...
+@inproceedings{Takahashi:2019,
+    doi = {10.21437/interspeech.2019-1550},
+    url = {https://doi.org/10.21437%2Finterspeech.2019-1550},
+    year = 2019,
+    month = {sep},
+    publisher = {{ISCA}},
+    author = {Takahashi, N. and Parthasaarathy, S. and Goswami, N. and Mitsufuji, Y.},
+    title = {Recursive Speech Separation for Unknown Number of Speakers},
+    booktitle = {Interspeech 2019}
+}
```
Comment (Emrys365) [inline doc/paper/espnet-se++/paper.bib:80]:
```suggestion
@inproceedings{Takahashi:2019,
    doi = {10.21437/interspeech.2019-1550},
    url = {https://doi.org/10.21437%2Finterspeech.2019-1550},
    pages={1348--1352},
    year = 2019,
    month = {sep},
    publisher = {{ISCA}},
    author = {Takahashi, N. and Parthasaarathy, S. and Goswami, N. and Mitsufuji, Y.},
    title = {Recursive Speech Separation for Unknown Number of Speakers},
    booktitle = {Interspeech 2019}
}
```

File: doc/paper/espnet-se++/paper.bib [hunk 4]
```diff
@@ -7,17 +8,156 @@ @inproceedings{Li:2021
...
+@inproceedings{Luo:2019a,
+    doi = {10.1109/asru46091.2019.9003849},
+    url = {https://doi.org/10.1109%2Fasru46091.2019.9003849},
+    year = 2019,
+    month = {dec},
+    publisher = {{IEEE}},
+    author = {Luo, Y. and Han, C. and Mesgarani, N. and Ceolini, E. and Liu, S.},
+    title = {{FaSNet}: Low-Latency Adaptive Beamforming for Multi-Microphone Audio Processing},
+    booktitle = {2019 {IEEE} Automatic Speech Recognition and Understanding Workshop ({ASRU})}
+}
```
Comment (Emrys365) [inline doc/paper/espnet-se++/paper.bib:91]:
```suggestion
@inproceedings{Luo:2019a,
    doi = {10.1109/asru46091.2019.9003849},
    url = {https://doi.org/10.1109%2Fasru46091.2019.9003849},
    pages={260--267},
    year = 2019,
    month = {dec},
    publisher = {{IEEE}},
    author = {Luo, Y. and Han, C. and Mesgarani, N. and Ceolini, E. and Liu, S.},
    title = {{FaSNet}: Low-Latency Adaptive Beamforming for Multi-Microphone Audio Processing},
    booktitle = {2019 {IEEE} Automatic Speech Recognition and Understanding Workshop ({ASRU})}
}
```

File: doc/paper/espnet-se++/paper.bib [hunk 5]
```diff
@@ -7,17 +8,156 @@ @inproceedings{Li:2021
...
+@inproceedings{Lu:2022a,
+    doi = {10.1109/icassp43922.2022.9747146},
+    url = {https://doi.org/10.1109%2Ficassp43922.2022.9747146},
+    year = 2022,
+    month = {may},
+    publisher = {{IEEE}},
+    author = {Lu, Y. J. and Cornell, S. and Chang, X. and Zhang, W. and Li, C. and Ni, Z. and Wang, Z. and Watanabe, S.},
+    title = {Towards Low-Distortion Multi-Channel Speech Enhancement: The {ESPNET}-Se Submission to the L3DAS22 Challenge},
+    booktitle = {{ICASSP} 2022 - 2022 {IEEE} International Conference on Acoustics, Speech and Signal Processing ({ICASSP})}
+}
```
Comment (Emrys365) [inline doc/paper/espnet-se++/paper.bib:102]:
```suggestion
@inproceedings{Lu:2022a,
    doi = {10.1109/icassp43922.2022.9747146},
    url = {https://doi.org/10.1109%2Ficassp43922.2022.9747146},
    pages={9201--9205},
    year = 2022,
    month = {may},
    publisher = {{IEEE}},
    author = {Lu, Y. J. and Cornell, S. and Chang, X. and Zhang, W. and Li, C. and Ni, Z. and Wang, Z. and Watanabe, S.},
    title = {Towards Low-Distortion Multi-Channel Speech Enhancement: The {ESPNET}-Se Submission to the {L3DAS22} Challenge},
    booktitle = {2022 {IEEE} International Conference on Acoustics, Speech and Signal Processing ({ICASSP})}
}
```

File: doc/paper/espnet-se++/paper.bib [hunk 6]
```diff
@@ -7,17 +8,156 @@ @inproceedings{Li:2021
...
+@inproceedings{Luo:2018,
+    doi = {10.1109/icassp.2018.8462116},
+    url = {https://doi.org/10.1109%2Ficassp.2018.8462116},
+    year = 2018,
+    month = {apr},
+    publisher = {{IEEE}},
+    author = {Luo, Y. and Mesgarani, N.},
+    title = {{TaSNet}: Time-Domain Audio Separation Network for Real-Time, Single-Channel Speech Separation},
+    booktitle = {2018 {IEEE} International Conference on Acoustics, Speech and Signal Processing ({ICASSP})}
+}
```
Comment (Emrys365) [inline doc/paper/espnet-se++/paper.bib:113]:
```suggestion
@inproceedings{Luo:2018,
    doi = {10.1109/icassp.2018.8462116},
    url = {https://doi.org/10.1109%2Ficassp.2018.8462116},
    pages={696--700},
    year = 2018,
    month = {apr},
    publisher = {{IEEE}},
    author = {Luo, Y. and Mesgarani, N.},
    title = {{TaSNet}: Time-Domain Audio Separation Network for Real-Time, Single-Channel Speech Separation},
    booktitle = {2018 {IEEE} International Conference on Acoustics, Speech and Signal Processing ({ICASSP})}
}
```

File: doc/paper/espnet-se++/paper.bib [hunk 7]
```diff
@@ -7,17 +8,156 @@ @inproceedings{Li:2021
...
+@inproceedings{Le:2019,
+    doi = {10.1109/icassp.2019.8683855},
+    url = {https://doi.org/10.1109%2Ficassp.2019.8683855},
+    year = 2019,
+    month = {may},
+    publisher = {{IEEE}},
+    author = {Le Roux, J. and Wisdom, S. and Erdogan, H. and Hershey, J. R.},
+    title = {{SDR} {\textendash} Half-baked or Well Done?},
+    booktitle = {{ICASSP} 2019 - 2019 {IEEE} International Conference on Acoustics, Speech and Signal Processing ({ICASSP})}
+}
```
Comment (Emrys365) [inline doc/paper/espnet-se++/paper.bib:124]:
```suggestion
@inproceedings{Le:2019,
    doi = {10.1109/icassp.2019.8683855},
    url = {https://doi.org/10.1109%2Ficassp.2019.8683855},
    pages={626--630},
    year = 2019,
    month = {may},
    publisher = {{IEEE}},
    author = {Le Roux, J. and Wisdom, S. and Erdogan, H. and Hershey, J. R.},
    title = {{SDR} {\textendash} Half-baked or Well Done?},
    booktitle = {2019 {IEEE} International Conference on Acoustics, Speech and Signal Processing ({ICASSP})}
}
```

File: doc/paper/espnet-se++/paper.bib [hunk 8]
```diff
@@ -7,17 +8,156 @@ @inproceedings{Li:2021
...
+@inproceedings{Boeddeker:2021,
+    doi = {10.1109/icassp39728.2021.9414661},
+    url = {https://doi.org/10.1109%2Ficassp39728.2021.9414661},
+    year = 2021,
+    month = {jun},
+    publisher = {{IEEE}},
+    author = {Boeddeker, C. and Zhang, W. and Nakatani, T. and Kinoshita, K. and Ochiai, T. and Delcroix, M. and Kamo, N. and Qian, Y. and Haeb-Umbach, R.},
+    title = {Convolutive Transfer Function Invariant {SDR} Training Criteria for Multi-Channel Reverberant Speech Separation},
+    booktitle = {{ICASSP} 2021 - 2021 {IEEE} International Conference on Acoustics, Speech and Signal Processing ({ICASSP})}
+}
```
Comment (Emrys365) [inline doc/paper/espnet-se++/paper.bib:135]:
```suggestion
@inproceedings{Boeddeker:2021,
    doi = {10.1109/icassp39728.2021.9414661},
    url = {https://doi.org/10.1109%2Ficassp39728.2021.9414661},
    pages={8428--8432},
    year = 2021,
    month = {jun},
    publisher = {{IEEE}},
    author = {Boeddeker, C. and Zhang, W. and Nakatani, T. and Kinoshita, K. and Ochiai, T. and Delcroix, M. and Kamo, N. and Qian, Y. and Haeb-Umbach, R.},
    title = {Convolutive Transfer Function Invariant {SDR} Training Criteria for Multi-Channel Reverberant Speech Separation},
    booktitle = {2021 {IEEE} International Conference on Acoustics, Speech and Signal Processing ({ICASSP})}
}
```

File: doc/paper/espnet-se++/paper.bib [hunk 9]
```diff
@@ -7,17 +8,156 @@ @inproceedings{Li:2021
...
+@inproceedings{Scheibler:2022,
+    doi = {10.1109/icassp43922.2022.9747473},
+    url = {https://doi.org/10.1109%2Ficassp43922.2022.9747473},
+    year = 2022,
+    month = {may},
+    publisher = {{IEEE}},
+    author = {Scheibler, R.},
+    title = {{SDR} {\textemdash} Medium Rare with Fast Computations},
+    booktitle = {{ICASSP} 2022 - 2022 {IEEE} International Conference on Acoustics, Speech and Signal Processing ({ICASSP})}
+}
```
Comment (Emrys365) [inline doc/paper/espnet-se++/paper.bib:146]:
```suggestion
@inproceedings{Scheibler:2022,
    doi = {10.1109/icassp43922.2022.9747473},
    url = {https://doi.org/10.1109%2Ficassp43922.2022.9747473},
    pages={701--705},
    year = 2022,
    month = {may},
    publisher = {{IEEE}},
    author = {Scheibler, R.},
    title = {{SDR} {\textemdash} Medium Rare with Fast Computations},
    booktitle = {2022 {IEEE} International Conference on Acoustics, Speech and Signal Processing ({ICASSP})}
}
```

File: doc/paper/espnet-se++/paper.bib [hunk 10]
```diff
@@ -84,6 +229,7 @@ @article{Ravanelli:2021
 }
 
 @inproceedings{Povey:2011,
+  doi = {10.15199/48.2016.11.70},
   title={The Kaldi speech recognition toolkit},
```
Comment (Emrys365) [inline doc/paper/espnet-se++/paper.bib:233]:
```suggestion
  title={The {Kaldi} speech recognition toolkit},
```

File: doc/paper/espnet-se++/paper.bib [hunk 11]
```diff
@@ -94,6 +240,7 @@ @inproceedings{Povey:2011
 
 
 @inproceedings{Luo:2020,
+  doi = {10.1109/icassp40776.2020.9054266},
   title={Dual-path rnn: efficient long sequence modeling for time-domain single-channel speech separation},
```
Comment (Emrys365) [inline doc/paper/espnet-se++/paper.bib:244]:
```suggestion
  title={Dual-path {RNN}: efficient long sequence modeling for time-domain single-channel speech separation},
```

File: doc/paper/espnet-se++/paper.bib [hunk 12]
```diff
@@ -102,16 +249,8 @@ @inproceedings{Luo:2020
...
-  booktitle={2018 IEEE International Conference on Acoustics, Speech and Signal Processing (ICASSP)},
-  pages={696--700},
-  year={2018},
-  organization={IEEE}
-}
-
-@article{Luo:2019,
+@article{Luo:2019b,
+  doi = {10.1109/taslp.2019.2915167},
   title={Conv-tasnet: Surpassing ideal time--frequency magnitude masking for speech separation},
```
Comment (Emrys365) [inline doc/paper/espnet-se++/paper.bib:254]:
```suggestion
  title={{Conv-TasNet}: Surpassing ideal time--frequency magnitude masking for speech separation},
```

File: doc/paper/espnet-se++/paper.bib [hunk 13]
```diff
@@ -7,17 +8,156 @@ @inproceedings{Li:2021
...
+  author={Hu, Y. and Liu, Y. and Lv, S. and Xing, M. and Zhang, S. and Fu, Y. and Wu, J. and Zhang, B. and Xie, L.},
+  journal={arXiv preprint arXiv:2008.00264},
+  year={2020}
+}
+
+@article{Tan:2021,
+  doi = {10.1109/taslp.2021.3082318},
+	title={Deep learning based real-time speech enhancement for dual-microphone mobile phones},
+  author={Tan, K. and Zhang, X. and Wang, D.},
+  journal={IEEE/ACM transactions on audio, speech, and language processing},
```
Comment (Emrys365) [inline doc/paper/espnet-se++/paper.bib:43]:
```suggestion
  journal={IEEE/ACM Transactions on Audio, Speech, and Language Processing},
```

File: doc/paper/espnet-se++/paper.bib [hunk 14]
```diff
@@ -102,16 +249,8 @@ @inproceedings{Luo:2020
...
-  year={2018},
-  organization={IEEE}
-}
-
-@article{Luo:2019,
+@article{Luo:2019b,
+  doi = {10.1109/taslp.2019.2915167},
   title={Conv-tasnet: Surpassing ideal time--frequency magnitude masking for speech separation},
   author={Luo, Y. and Mesgarani, N.},
   journal={IEEE/Association for Computing Machinery (ACM) transactions on audio, speech, and language processing},
```
Comment (Emrys365) [inline doc/paper/espnet-se++/paper.bib:256]:
```suggestion
  journal={IEEE/ACM Transactions on Audio, Speech, and Language Processing},
```

File: doc/paper/espnet-se++/paper.bib [hunk 15]
```diff
@@ -163,6 +296,7 @@ @inproceedings{Yu:2017
 }
 
 @article{Towns:2014,
+  doi = {10.1109/mcse.2014.80},
   title={XSEDE: accelerating scientific discovery},
```
Comment (Emrys365) [inline doc/paper/espnet-se++/paper.bib:300]:
```suggestion
  title={{XSEDE}: accelerating scientific discovery},
```

File: doc/paper/espnet-se++/paper.bib [hunk 16]
```diff
@@ -163,6 +296,7 @@ @inproceedings{Yu:2017
...
   author={Towns, J. and Cockerill, T. and Dahan, M. and Foster, I. and Gaither, K. and Grimshaw, A. and Hazlewood, V. and Lathrop, S. and Lifka, D. and Peterson, G. D. and Roskies, R. and Scott, J. R. and Wilkins-Diehr, N.},
   journal={Computing in science \& engineering},
```
Comment (Emrys365) [inline doc/paper/espnet-se++/paper.bib:302]:
```suggestion
  journal={Computing in Science \& Engineering},
```

File: doc/paper/espnet-se++/paper.bib [hunk 17]
```diff
@@ -174,6 +308,7 @@ @article{Towns:2014
 }
 
 @inproceedings{Nystrom:2015,
+  doi = {10.1145/2792745.2792775},
   title={Bridges: a uniquely flexible HPC resource for new communities and data analytics},
```
Comment (Emrys365) [inline doc/paper/espnet-se++/paper.bib:312]:
```suggestion
  title={Bridges: a uniquely flexible {HPC} resource for new communities and data analytics},
```

File: doc/paper/espnet-se++/paper.bib [hunk 18]
```diff
@@ -94,6 +240,7 @@ @inproceedings{Povey:2011
 
 
 @inproceedings{Luo:2020,
+  doi = {10.1109/icassp40776.2020.9054266},
   title={Dual-path rnn: efficient long sequence modeling for time-domain single-channel speech separation},
   author={Luo, Y. and Chen, Z. and Yoshioka, T.},
   booktitle={ICASSP 2020-2020 IEEE International Conference on Acoustics, Speech and Signal Processing (ICASSP)},
```
Comment (Emrys365) [inline doc/paper/espnet-se++/paper.bib:246]:
```suggestion
  booktitle={2020 IEEE International Conference on Acoustics, Speech and Signal Processing (ICASSP)},
```

**Comment by Emrys365** [comment]
@faroit Sorry for my late response! I approve the above comments with some formatting suggestions.

**Comment by sw005320** [comment]
@Emrys365, how about committing your suggestions by yourself (They look good to me)?
Then, we can merge this PR.

**Comment by Emrys365** [comment]
OK, I have committed the changes.

**Comment by neillu23** [comment]
@Emrys365 I appreciate your assistance! My apologies for overlooking your earlier remarks.

**Pull request merged**